src/course/syllabus/application/feature/learning-outcomes.md

```diff
@@ -0,0 +1,38 @@
+## Planning
```

Comment:
add something around being able to look up and learn the right js features/methods to implement things.